cast/src/main.rs

```diff
@@ -65,12 +70,15 @@ async fn main() -> Result<()> {
 
     match cli.command {
         Commands::Declare(declare) => {
+            let parent = accounts_file_path.parent().unwrap();
+            if !accounts_file_path.exists() {
+                anyhow::bail!("accounts file ({}) does not exist", parent);
+            }
```

Comment:
Maybe this check should be applicable to other commands as well, and moved out of `match`?

Reply:
yes, good point, thx!

Reply:
[move account file check outside of match case](https://github.com/software-mansion/protostar/pull/2138/commits/f2c1dc2f40a37b6a7b6bc75942ee051d84aac23e)